src/common/AbstractObject.h

```diff
@@ -1068,6 +1068,12 @@ namespace COMMON_NS
 		 */
 		DLL_IMPORT_OR_EXPORT std::string getExtraMetadataStringValueAtIndex(unsigned int index) const;
 		
+		/**
+		* Build and return an URI from an Energistics object.
```

Comment:
"ETP1.2 URI" would be better than just "URI" since URI is very generic.
"Energistics object" are more and more called "Energistics dataobject".

Reply:
Done.